Nodejs/Tests/Core.UI/BasicIntellisense.cs

```diff
@@ -45,6 +45,7 @@ public sealed class BasicIntellisense : NodejsProjectTest {
 
                 server.MoveCaret(2, 13);
                 solution.ExecuteCommand("Edit.CompleteWord");
+                Keyboard.Type("\t");
```

Comment:
Edit.CompleteWord is not doing what we think it is. Looking at [IntellisenseControler.cs#L646](https://github.com/Microsoft/nodejstools/blob/5c994768c5678579b76d71ba6ff48a30fa5991cd/Nodejs/Product/Nodejs/Intellisense/IntellisenseController.cs#L646), it does the same thing with ShowMemberList.